### PR TITLE
Cuquantum added to run on batch

### PR DESCRIPTION
**Pull request by jrossthomson**
combined code into single .py file. Added support for proper gcs_fuse mounts.

### CODE REVIEW

**Comment by wkharold** [comment]
LGTM

**Review by wkharold — APPROVED** [review]

File: cloud-batch-python/python-batch/cuquantum-noise.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+project_id: "gromacs-demo-jrt"
```
Comment (mmm) [inline cloud-batch-python/python-batch/cuquantum-noise.yaml:1]: do we want a project name here?

File: cloud-batch-python/python-batch/deepvariant.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+project_id: "gromacs-demo-jrt"
```
Comment (mmm) [inline cloud-batch-python/python-batch/deepvariant.yaml:1]: ditto

File: cloud-batch-python/python-batch/nvidia-smi-container.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+project_id: "gromacs-demo-jrt"
```
Comment (mmm) [inline cloud-batch-python/python-batch/nvidia-smi-container.yaml:1]: ditto

File: cloud-batch-python/python-batch/shell_with_gcsfuse.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+project_id: "gromacs-demo-jrt"
```
Comment (mmm) [inline cloud-batch-python/python-batch/shell_with_gcsfuse.yaml:1]: ditto

**Review by mmm — COMMENTED** [review]
also everything needs copyright blurbs

**Review by mmm — APPROVED** [review]

**Pull request merged**